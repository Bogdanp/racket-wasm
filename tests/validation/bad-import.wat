(module
 (type (func))
 (type (func (param i32) (result i32)))
 (import "a" "m" (memory 0))
 (import "a" "f" (func $f (type 1)))
 (func
  (call $f)))