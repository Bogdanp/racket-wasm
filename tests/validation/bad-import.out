(#f . "arity error\n  expected: [i32]\n  found: []\n  at: (instr:call 0)\n  within: (funcidx 0)")